lib/android-helpers.js

```diff
@@ -11,7 +11,6 @@ import Bootstrap from 'appium-android-bootstrap';
 import ADB from 'appium-adb';
 
 const REMOTE_TEMP_PATH = "/data/local/tmp";
-const REMOTE_INSTALL_TIMEOUT = 90000; // milliseconds
```

Comment:
This should remain, to be used as the default (as it stands, there is no default, so if the `adbInstallTimeout` is not given, what happens?).